src/exchange/exchange.service.ts

```diff
@@ -0,0 +1,82 @@
+import { Inject, Injectable, Logger } from "@nestjs/common";
+import { User } from "../data/entities/user.entity";
+import { Request } from "../data/entities/request.entity";
```

Comment:
Is there not an `index.ts` file for entities that lets you have these all on the same line?

Reply:
good point, but not gonna do it on this ticket, its a bug refactor already.